src/arrayMethodPush.js

```diff
@@ -5,8 +5,9 @@
  *
  */
 function applyCustomPush() {
-  [].__proto__.push2 = function() {
-    // write code here
+  [].__proto__.push2 = function(...args) {
+    this.splice(this.length, 0, ...args);
```

Comment:
Would be cool to do it without any array methods

Reply:
yeah, using array methods is not allowed for these tasks.

Reply:
I was wrong, this is allowed :)